include/libopencm3/usb/audio.h

```diff
@@ -6,10 +6,11 @@
 
 @version 1.0.0
 
-@author @htmlonly &copy; @endhtmlonly 2014
+@author @htmlonly &copy; @endhtmlonly 2018
 Daniel Thompson <daniel@redfelineninja.org.uk>
+Seb Holzapfel <schnommus@gmail.com>
 
-@date 19 April 2014
+@date 20 June 2018
```

Comment:
don't change this, you should be (if you must) _adding_ years, not dropping them

Reply:
no worries - reverted. (accidentally blindly changing dates...)

Reply:
also I'm having trouble getting doxygen to actually render this file so it's hard for me to check :(